tracker/dmlc_tracker/ssh.py

```diff
@@ -20,23 +21,67 @@ def sync_dir(local_dir, slave_node, slave_dir):
         slave_node[1], local_dir, remote)
     subprocess.check_call([prog], shell = True)
 
-def get_env(pass_envs):
+def prepare_envs(args):
+    """
+    Load environment variables from arguments
+    """
+    envs = {}
+    # default env variables passed
+    envs['default'] = {'OMP_NUM_THREADS', 'LD_LIBRARY_PATH', 'AWS_ACCESS_KEY_ID',
+                          'AWS_SECRET_ACCESS_KEY', 'DMLC_INTERFACE'}
```

Comment:
Why is KMP_AFFINITY removed from this list ?

Reply:
good catch. I added it back. I suspect it has to do with the fact that I was working on this before KMP_AFFINITY was added. Either way, brought it back.